docs/content/node-operation/past-sporks.mdx

```diff
@@ -249,7 +249,7 @@ While FCL v0.0.67 is not strictly necessary to use Mainnet 6, we strongly recomm
 | **Root Parent ID**: 5bc2b0900a5138e39d9209a8fe32e14b3e5c884bd36d2a645620f746b7c8bd47   |
 | **Root State Commit**: 6b9161a225b087a461ec95e710fdf4e73f6d6c9401ebf066207a021dced4ce5e |
 | **Git Commit**:                                                                        |
-| **Branch/Tag**: v0.8.2                                                                 |
```

Comment:
why were these removed?

Reply:
I removed it because I couldn't find either a branch or a tag that has this version.